content/posts/newsletter-013/index.md

```diff
@@ -124,6 +124,62 @@ Chillscapes is written using an early-in-development 2d engine,
 [neoc]: https://itch.io/jam/neoc03-rhythm-jam
 [kludgine]: https://github.com/khonsulabs/kludgine
 
+### [Veloren][veloren]
+
+![Landscape](veloren-landscape1.png)
+
+^ _Landscape with new LoD and lighting_
```

Comment:
Some CSS has been added in #264 to make captions format nicely, so you don't need the arrow any more - just make sure to not leave a gap between the image and the caption, or markdown will split them into seperate paragraphs

```suggestion
_Landscape with new LoD and lighting_
```